src/main/java/lesson6/task3/CharCounter.java

```diff
@@ -0,0 +1,16 @@
+package lesson6.task3;
+
+import lesson6.task0.FileUtils;
+
+import java.io.IOException;
+
+public class CharCounter {
+
+    public static int countChars() throws IOException {
+        String filePath = "src/main/resources/data.txt";
```

Comment:
file path in method param

Reply:
Parameter included and tests adjusted. Please review.